### ISSUE TITLE
crest 2.12 example-2 test fails with threads > 1 -- SOLVED

### ISSUE THREAD

**Issue by rjrich** [grimme-lab/xtb#751]
Installed crest 2.12 by downloading the precompiled binary. Running tests from the crest 2.12 source file. Passed expl-0 and expl-1, but expl-2 failed with the following message:

"xTB Geometry Optimization
Initial geometry optimization failed!
Please check your input."

However, if I revise the run.sh file by changing "-T 4" to "-T 1", the job completes normally with no errors.

Here are my machine parameters:

CPU = AMD TR Pro 5975wx, 32 cores/64 threads,
RAM = 256GB
GPU = 2 x Nvidia RTX A-6000
OS = Linux Mint 21.1 Xfce (based on Ubuntu 22.04 LTS)

Thank you for your assistance.

**Comment by rjrich** [comment]
Update: I removed crest and xtb from the system. I compiled xtb from source, which was successful. I tried compiling crest 2.12 and 3.0dev from source, but these attempts failed after trying all available methods (e.g., with ifort and icc, as well as with gfortran and gcc; with meson and ninja, as well as with cmake and make. Finally, I downloaded the pre-compiled version of crest.

I then ran all of the examples in the crest-2.12 source download, and this time, all tests passed.

Therefore, although I would have liked to have been successful compiling crest from source, I think this issue has been SOLVED.

Thank you.
